src/modules/Telemetry/EnvironmentTelemetry.cpp

```diff
@@ -54,6 +55,7 @@ AHT10Sensor aht10Sensor;
 MLX90632Sensor mlx90632Sensor;
 DFRobotLarkSensor dfRobotLarkSensor;
 NAU7802Sensor nau7802Sensor;
+extern BMP3XXSensor bmp3xxSensor;
 #ifdef T1000X_SENSOR_EN
```

Comment:
why the extermal reference?

Reply:
This is fixed now - the reason was BMP388 sensor really should be accessed through only one instance to avoid conflicting calls. The latest commit to this PR adds a singleton wrapper around the the underlying BMP3XXSensor class to sort this properly